mlflow/server/js/src/experiment-tracking/components/modals/CreateExperimentModal.tsx

```diff
@@ -24,7 +24,7 @@ type CreateExperimentModalImplProps = {
   onClose: (...args: any[]) => any;
   experimentNames: string[];
   createExperimentApi: (...args: any[]) => any;
-  searchExperimentsApi: (...args: any[]) => any;
+  invalidate: () => void;
```

Comment:
nit: for this kind of callback pattern, we usually use more specific names (e.g. `onCreatedExperiment`)